borg/archive.py

```diff
@@ -786,11 +791,11 @@ def add_callback(chunk):
 
         def add_reference(id_, size, csize, cdata=None):
             try:
-                count, _, _ = self.chunks[id_]
-                self.chunks[id_] = count + 1, size, csize
+                entry = self.chunks[id_]
+                self.chunks[id_] = entry._replace(refcount=entry.refcount + 1, size=size, csize=csize)
             except KeyError:
                 assert cdata is not None
-                self.chunks[id_] = 1, size, csize
+                self.chunks[id_] = ChunkIndexEntry(refcount=1, size=size, csize=csize)
                 if self.repair:
                     self.repository.put(id_, cdata)
```

Comment:
yes, it misses the "repair" functionality, so I am keeping it as is (this is not in the scope of the namedtuple PR anyway).